storage-resize-images/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Version 0.1.3
+
+feature - Automatically copy the following metadata, if present, from the original image to the resized image(s): `Cache-Control`, `Content-Disposition`, `Content-Encoding`, `Content-Language`, `Content-Type`, and user-provided metadata (except Firebase storage download tokens).
```

Comment:
We should include that if Cache-Control param is configured, the value will overwrite the copied value.  I'd add the following as a final sentence to this bullet.

Note that you can configure your extension to overwrite the `Cache-Control` value for the resized image(s).

Reply:
Done!